tests/functional/conftest.py

```diff
@@ -118,3 +119,9 @@ def sender(test_accounts):
 @pytest.fixture
 def receiver(test_accounts):
     return test_accounts[1]
+
+
+@pytest.fixture
+def test_provider():
```

Comment:
You can use the existing fixture `eth_tester_provider`

Reply:
I will fix this